symphony/bdk/core/config/model/bdk_certificate_config.py

```diff
@@ -21,3 +21,20 @@ def is_valid(self) -> bool:
         :return: true if the certificate configuration is valid.
         """
         return not (self.path is not None and self.content != "")
+
+    def setContent(self, certificate_content):
```

Comment:
Please follow python conventions: https://www.python.org/dev/peps/pep-0008/#naming-conventions
Function names should be lowercase with underscores.

Reply:
Cannot we just use override the setter for this? (to make sure we have a content or a path but not both)